internal/business/server/healthcheck_handler.go

```
@@ -21,6 +21,41 @@ import (
+}
+
+func maskURLs(response map[string]any) map[string]any {
+	masked := make(map[string]any, len(response))
+	for k, v := range response {
+		switch val := v.(type) {
+		case []*healthcheck.Response:
+			maskedList := make([]*healthcheck.Response, len(val))
+			for i, r := range val {
+				maskedList[i] = maskResponse(r)
+			}
+			masked[k] = maskedList
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)